hoodie-common/pom.xml

```diff
@@ -121,6 +121,10 @@
       <artifactId>jackson-core-asl</artifactId>
       <version>1.9.13</version>
     </dependency>
+    <dependency>
```

Comment:
hoodie-common cannot depend on spark.. this is also pulled in by hoodie-hadoop-mr.. Please remove this dependency.

Reply:
removed it. Earlier we had created new HoodieSparkTaskContext to pass on TaskContext information to newly launched thread (as TaskContext.get() is threadlocal). But now we have figured out another way to set TaskContext which will not need any of these changes. "TaskContext$.MODULE$.setTaskContext(taskContext);"